rosidl_generator_rs/cmake/rosidl_generator_rs_generate_interfaces.cmake

```diff
@@ -130,6 +130,11 @@ rosidl_write_generator_arguments(
   TARGET_DEPENDENCIES ${target_dependencies}
 )
 
+file(READ ${generator_arguments_file} contents)
```

Comment:
Oh, I just realized that `rosidl_write_generator_arguments` can't pass the package version, I'll file a ticket in the `rosidl` repo, but for now this code is fine. Thanks!